app/shared/date-picker/DatePicker.js

```diff
@@ -0,0 +1,37 @@
+import React, { PropTypes } from 'react';
+import moment from 'moment';
+import { DateField, DatePicker as RDPDatePicker } from 'react-date-picker';
+
+const dateFormat = 'YYYY-MM-DD';
+const localizedDateFormat = 'D.M.YYYY';
+
+function formatDate(date) {
+  return moment(date, localizedDateFormat).format(dateFormat);
+}
+
+DatePicker.propTypes = {
+  onChange: PropTypes.func.isRequired,
+  value: PropTypes.string.isRequired,
+};
+
+function DatePicker(props) {
+  return (
+    <DateField
+      className="date-picker"
+      clearIcon={false}
+      collapseOnDateClick
+      dateFormat={localizedDateFormat}
+      footer={false}
+      onChange={date => props.onChange({ date: formatDate(date) })}
```

Comment:
- [ ] If I was using this component I'd expect the argument to onChange to be either the change event or the new value, not necessarily an object that contains the new value in a `date` attribute. So I think it'd make more sense for this to be `onChange={props.onChange}` or `onChange={date => props.onChange(formatDate(date))}`.